src/System.Management.Automation/engine/CommandSearcher.cs

```diff
@@ -1389,12 +1389,8 @@ internal LookupPathCollection ConstructSearchPatternsFromName(string name, bool
         /// True if the command name is either a provider-qualified or PowerShell drive-qualified
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By Process Id.Can enter using NamedPipeConnectionInfo
```powershell
Expected $true, but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 148
148:                     Should -BeTrue -Because (get-content $splat.FilePath -Raw)
```